.github/copilot-instructions.md

```diff
@@ -0,0 +1,157 @@
+# LKG Halle Church Event Management System
+
+LKG Halle is a Node.js-based church event management system with automated calendar synchronization, website updates, and YouTube broadcast creation. The project consists of automation scripts in the root directory and an Astro-based website in the `/website/` subfolder.
+
+**Always reference these instructions first and fallback to search or bash commands only when you encounter unexpected information that does not match the info here.**
+
+## Working Effectively
+
+### Bootstrap and Dependencies
+
+- Install Node.js dependencies:
+  - Root project: `npm install` -- takes 2 minutes. NEVER CANCEL. Set timeout to 5+ minutes.
+  - Website: `cd website && npm install` -- takes 34 seconds. NEVER CANCEL. Set timeout to 2+ minutes.
```

Comment:
Please use `npm ci` instead. We don't want the lockfile to change unnecessariy. Only use install if you need to install new dependencies